packages/common/src/api/tan-query/coins/useCoin.ts

```diff
@@ -0,0 +1,54 @@
+import { useQuery } from '@tanstack/react-query'
+
+import { useQueryContext } from '~/api/tan-query/utils'
+
+import { QUERY_KEYS } from '../queryKeys'
+import { QueryKey, QueryOptions } from '../types'
+
+export interface CoinInfo {
+  ticker: string
+  mint: string
+  user_id: string
+  created_at: string
+  members: number
+  members_24h_change_percent: number
+  token_info: {
+    price: number
+    price_change_24h: number
+    market_cap: number
+    volume_24h: number
+    supply: number
+    holders: number
+  }
```

Comment:
can be broken out into its own type

Reply:
Oh yeah good catch actually we don't need this hook at all rn